catalyst-gateway/bin/src/service/api/documents/templates/data.rs

```diff
@@ -107,7 +113,10 @@ impl From<CommentTemplateDocData> for SignedDocData {
         Self {
             id: value.0,
             ver: value.0,
-            doc_type: catalyst_signed_doc::doc_types::COMMENT_TEMPLATE_UUID_TYPE,
+            doc_type: catalyst_signed_doc::doc_types::PROPOSAL_COMMENT
```

Comment:
No, can't have an `expect` in this place, change the `doc_type` from `Uuid` to `DocType`